src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/DocGenService.java

```diff
@@ -283,6 +283,10 @@ void replaceText(final DocumentTemplateSource documentTemplateSource,
       {
          dataReplacement = PLACEHOLDER_UNKNOWN;
       }
+      if(item.getText().equals("«Insert Time, Date, Month, Year of #1»") && !isPublish)
```

Comment:
Hope this is tested and make sure this date is a publish date for all agreements